src/main/java/teammates/ui/controller/InstructorFeedbacksPageData.java

```diff
@@ -280,6 +280,15 @@ public FeedbackSessionsCopyFromModal getCopyFromModal() {
     }
 
     /**
+     * Retrieves the link to submit the request to remind a particular student(s)
+     * Also contains home page link to return after the action.
+     * @return form submit action link
+     */
+    public String getRemindParticularStudentsLink() {
+        return getInstructorFeedbackRemindParticularStudentsLink(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE);
```

Comment:
`* Also contains home page link to return after the action.`

In general, do go through all the comments you added and ensure they are correct.
"a particular student(s)" can be simply represented as "particular students".